java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -26,18 +26,6 @@ public abstract class BaseClient implements AutoCloseable {
     protected final ConnectionManager connectionManager;
     protected final CommandManager commandManager;
 
-    /**
-     * Extracts the response from the Protobuf response and either throws an exception or returns the
-     * appropriate response as an Object
-     *
-     * @param response Redis protobuf message
-     * @return Response Object
-     */
-    protected Object handleObjectResponse(Response response) {
```

Comment:
Moving this below.  There should be about a dozen handleXResponse functions to be included in upcoming PRs and they should be below the CreateClient calls.